extern/storage-sealing/input.go

```diff
@@ -400,6 +400,9 @@ func (m *Sealing) tryCreateDealSector(ctx context.Context, sp abi.RegisteredSeal
 		return xerrors.Errorf("initializing sector: %w", err)
 	}
 
+	// update stats early, fsm planner would do that async
+	m.stats.updateSector(cfg, m.minerSectorID(sid), UndefinedSectorState)
```

Comment:
Without this we could end up creating a bunch of sectors at once, and start more parallel work than specified by the max sealing sector config.